site/docs/tools/cli.md

```diff
@@ -158,38 +187,64 @@ Usage: nessie log [OPTIONS] [REVISION_RANGE] [PATHS]...
+  <start_hash>..<end_hash> only show log for given range on the particular ref
+  that was provided
 
-  PATHS optional list of paths. If given, only show commits which affected
-  the given paths
+  PATHS optional list of paths. If given, only show commits which affected the
+  given paths
 
 Options:
-  -n, --number INTEGER    number of log entries to return
-  --since, --after TEXT   Commits more recent than specific date
-  --until, --before TEXT  Commits older than specific date
-  --author, --committer   limit commits to specific committer
-  --help                  Show this message and exit.
+  -r, --ref TEXT                  branch to list from. If not supplied the
+                                  default branch from config is used
+  -n, --number INTEGER            number of log entries to return
+  --since, --after TEXT           Only include commits newer than specific
+                                  date
+  --until, --before TEXT          Only include commits older than specific
+                                  date
+  --author TEXT                   Limit commits to a specific author (this is
+                                  the original committer). Supports specifying
+                                  multiple authors to filter by.
+  --committer TEXT                Limit commits to a specific committer (this
+                                  is the logged in user/account who performed
+                                  the commit). Supports specifying multiple
+                                  committers to filter by.
+  --query, --query-expression TEXT
+                                  Allows advanced filtering using the Common
+                                  Expression Language (CEL). An intro to CEL
+                                  can be found at
+                                  https://github.com/google/cel-
+                                  spec/blob/master/doc/intro.md. Some examples
+                                  with usable variables 'commit.author'
+                                  (string) / 'commit.committer' (string) /
+                                  'commit.commitTime' (timestamp) /
+                                  'commit.hash' (string) / 'commit.message'
+                                  (string) / 'commit.properties' (map) are:
+                                  commit.author=='nessie_author'
+                                  commit.committer=='nessie_committer'
+                                  timestamp(commit.commitTime) >
+                                  timestamp('2021-06-21T10:39:17.977922Z')
+  --help                          Show this message and exit.
 ```
 
 ``` bash
 $ nessie merge --help
 ``` 
 
 ``` bash
-Usage: nessie merge [OPTIONS] [MERGE_BRANCH]
+Usage: nessie merge [OPTIONS] [FROM_BRANCH]
 
-  Merge BRANCH into current branch. BRANCH can be a hash or branch.
+  Merge FROM_BRANCH into current branch. FROM_BRANCH can be a hash or branch.
```

Comment:
"Current" is confusing here. It either merged into the default branch or the explicitly specified branch. Maybe just use "another"?

Reply:
I actually like `Merge FROM_BRANCH into default branch or the explicitly specified branch` as this should make it quite clear what's meant